CONTRIBUTING.md

```diff
@@ -1,11 +1,51 @@
-# How to Contribute
+# Contributing to graphql-kimera
+
+> Please note that this project is released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
 
 Kimera is one of [Lola Tech's](https://www.lola.tech) first open source project that is both under very active development and is also being used to ship code to our enterprise customers.
 
-## Code of Conduct
+## Found an Issue?
+
+Thank you for reporting any issues you find. We do our best to test and make graphql-kimera as solid as possible, but any reported issue is a real help.
+
+> When raising an issue please use the template provided as a guideline, provided to you.
+
+## Want a new feature?
+
+> For raising a feature request please use the template provided as a guideline, provided to you.
```

Comment:
~template provided as a guideline, provided to you.~
provided to you as a guideline.